src/components/graph/graph.builder.js

```diff
@@ -164,11 +164,20 @@ function buildNodeProps(node, config, nodeCallbacks = {}, highlightedNode, highl
 
     const t = 1 / transform;
     const nodeSize = node.size || config.node.size;
+    var nodeSizeWidth = node.width || config.node.width;
+    var nodeSizeHeight = node.height || config.node.height;
```

Comment:
Please use `const`. Also, I would suggest a rename to `nodeHeight`.

Reply:
changed.